pkg/bpf/ringbuf_windows.go

```diff
@@ -285,7 +284,7 @@ func (reader *WindowsRingBufReader) Init(fd int, ring_buffer_size int) error {
 	var reply _ebpf_operation_map_query_buffer_reply
 	err = reader.invokeIoctl(unsafe.Pointer(&req), uint32(unsafe.Sizeof(req)), unsafe.Pointer(&reply), uint32(unsafe.Sizeof(reply)), nil)
 	if err != nil {
-		return fmt.Errorf("Failed to do device io control")
+		return fmt.Errorf("Failed to do device io control. Error = %v", err)
```

Comment:
Hey, convention in Go for error is lowercase, and wrap the next error next to the first one with `:`.
```suggestion
		return fmt.Errorf("failed to do device io control: %w", err)
```

Reply:
I ran golangci-lint on this branch, did not see the error.

Reply:
I have fixed this in latest commiyt